lua/astrocommunity/remote-development/distant-nvim/init.lua

```diff
@@ -0,0 +1,18 @@
+return {
+  "chipsenkbeil/distant.nvim",
+  branch = "v0.2",
+  cmd = {
+    "DistantLaunch",
+    "DistantOpen",
+    "DistantConnect",
+    "DistantInstall",
+    "DistantMetadata",
+    "DistantShell",
+    "DistantShell",
+    "DistantSystemInfo",
+    "DistantClientVersion",
+    "DistantSessionInfo",
+    "DistantCopy",
+  },
+  opts = function(_, opts) require("astronvim.utils").extend_tbl(opts, require("distant.settings").chip_default()) end,
```

Comment:
```suggestion
  opts = function(_, opts) opts["*"] = require("distant.settings").chip_default() end,
```

Shouldn't this be this? Based on your previous commit at least

Reply:
Sorry, oversight